docs/schema.md

```diff
@@ -559,10 +560,13 @@ properties:
     x-google-marketplace:
       type: SERVICE_ACCOUNT
       serviceAccount:
+        permissionExplanation: >
```

Comment:
Perhaps simply `description`?

I also think we should make this field required so that the UI always has something to show.

Reply:
Updated to `description` (hopefully the example is clear enough to distinguish from the property-level `description` field).

Checking within `Schema.validate()` allows the schema to be parsed without it, but not deployed (deployer calls `Schema.validate()`).